packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.component.tsx

```diff
@@ -58,6 +60,20 @@ const VitalsHeader: React.FC<VitalsHeaderProps> = ({ patientUuid }) => {
     const vitalsTakenToday = Boolean(dayjs(latestVitals?.date).isToday());
     const vitalsOverdue = hasActiveVisit && !vitalsTakenToday;
 
+    const now = dayjs();
+    const vitalsOverduePeriod = Math.round(dayjs.duration(now.diff(latestVitals?.date)).asDays());
+    let overdueVitalsTag: string = '';
+
+    if (vitalsOverduePeriod == 1) {
+      overdueVitalsTag = 'These vitals are 1 day old.';
+    } else if (vitalsOverduePeriod >= 2 && vitalsOverduePeriod <= 7) {
+      overdueVitalsTag = `These vitals are ${vitalsOverduePeriod} days old.`;
+    } else if (vitalsOverduePeriod >= 8 && vitalsOverduePeriod <= 14) {
+      overdueVitalsTag = 'These vitals are over one week old.';
+    } else {
+      overdueVitalsTag = 'These vitals are out of date.';
+    }
```

Comment:
```suggestion
    switch (true) {
      case vitalsOverduePeriod >= 1 && vitalsOverduePeriod <= 7:
        overdueVitalsTag = t('daysOldVitals', 'These vitals are {{vitalsOverduePeriod}} days old', {
          vitalsOverduePeriod,
        });
        break;
      case vitalsOverduePeriod >= 8 && vitalsOverduePeriod <= 14:
        overdueVitalsTag = t('overOneWeekOldVitals', 'These vitals are over one week old');
        break;
      default:
        overdueVitalsTag = t('outOfDateVitals', 'These vitals are out of date');
        break;
    }
```

This might work better according to the criteria laid out in the ticket:

- From 1 day and up to 7 days - ‘These vitals are X days old’
- Up to two weeks - ‘These vitals are over one week old’
- Over two weeks - ‘These vitals are out of date.'

Also, we need to translate those strings.